src/containers/categories.js

```diff
@@ -1,27 +1,47 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import * as actions from '../store/actions/index';
+
 import Layout from '../misc/layout';
 import CatGroup from '../components/category/catGroup';
 
 class Categories extends Component {
+
+  componentDidMount() {
+    this.props.getTypes();
+    this.props.getAlc();
+  }
+
+  filterDrinks(idCat,catName) {
```

Comment:
if this will be merged it should have console.logs, are you sure you are running linter to avoid this stuff?

Reply:
I'm not running linter yet, i'll use ESLint after this PR is that ok?

Reply:
Should I leave just the empty function without the console.logs? or delete the entire function and its reference inside the component?